systemd/systemd.go

```diff
@@ -1398,11 +1398,14 @@ var squashfsFsType = squashfs.FsType
 
 // Note that WantedBy=multi-user.target and Before=local-fs.target are
 // only used to allow downgrading to an older version of snapd.
-const regularMountUnitTmpl = `[Unit]
+//
+// We want (see isbeforedrivers) some snaps and components to be mounted before
```

Comment:
```suggestion
// We want (see isBeforeDrivers) some snaps and components to be mounted before
```
or simply drop the function name. Function names in comments have a tendency to become outdated and incorrect whenever refactors are introduced.

Reply:
Done, thanks